REFERENCE.md

```diff
@@ -9,6 +9,8 @@
 * [`profile_audit`](#profile_audit): A short summary of the purpose of this class
 * [`profile_audit::net_process_log`](#profile_auditnet_process_log): Logs user processes and open network connections to syslog
 * [`profile_audit::qualys`](#profile_auditqualys): Configure host to be scanned by qualys
+* [`profile_audit::qualys_eus_reporting`](#profile_auditqualys_eus_reporting): Allow qualys to sudo to root and add wrapper for subscription-manager to lie about enabled repos.
```

Comment:
Technically the sudo part is now separate, so this description is slightly wrong in how it is worded.

Reply:
Documentation updated